src/app/_graphql/allContent.ts

```diff
@@ -0,0 +1,56 @@
+import { MEDIA_FIELDS } from "@/app/_graphql/media";
+
+export const ALL_CONTENT = `
+query Content {
```

Comment:
There should probably be a limit here right? We don't want massive lists, and it could be a performance sink

Reply:
great point. updated ✅